packages/compiler-cli/test/ngcc/BUILD.bazel

```diff
@@ -10,6 +10,7 @@ ts_library(
     deps = [
         "//packages/compiler",
         "//packages/compiler-cli",
```

Comment:
I removed these two deps as they are not needed for testing ngcc right now and it makes the compilation more compilicated[sic]!